app/src/main/java/ch/epfl/polychef/users/User.java

```diff
@@ -149,4 +162,11 @@ public boolean equals(@Nullable Object obj) {
 
         return false;
     }
+
+    public static int getResourceImageFromActivity(HomePage hostActivity,User userToDisplay){
+        int profilePictureId=userToDisplay.getProfilePictureId();
+        String photoName=hostActivity.getResources().getStringArray(R.array.profilePicturesNames)[profilePictureId];
+        int resourceImage = hostActivity.getResources().getIdentifier(photoName, "drawable", hostActivity.getPackageName());
+        return resourceImage;
```

Comment:
You could use `GlobalApplication.getAppContext()` instead of passing the `HomePage hostActivity`, but if it works fine for now, we can refactor that later

Reply:
done, thanks